tests/defer/build.gradle.kts

```diff
@@ -13,6 +13,17 @@ kotlin {
     withJava()
   }
 
+  js {
+    nodejs {
+      testTask {
+        useMocha {
+          // Override default timeout (needed for subscriptions tests)
+          timeout = "120s"
```

Comment:
At this point, we could consider making this the default everywhere 😅

Reply:
Done! :)